src/Action/BaseAction.php

```diff
@@ -26,6 +26,15 @@
     protected $_responding = false;
 
     /**
+     * Default configuration
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'modelFactory' => 'Table',
```

Comment:
Hmm, right - worked in my case, but hadn't run the tests. Do you agree with the change? Should I go over all the actions?